articles-4.html

```diff
@@ -39,18 +39,23 @@ <h1>Skincare Articles</h1>
 				<!-- This is the category subheading that will change depending on which section of page user is on -->
 				<p class="subheading">Read articles dedicated to sensitive skin</p>
 				<div class="image-first">
-					<a href="#"><img src="sample.png"></a>
+					<a href="tips-for-sensitive-skin.html"><img src="images/sensitive/tips-for-sensitive-skin.jpg"></a>

```

Comment:
In this version I finished the last article page by adding the last 2 articles for my website and their images and links here.